src/legacy/core_plugins/kbn_vislib_vis_types/public/area.js

```diff
@@ -80,74 +92,42 @@ export default function PointSeriesVisType(Private) {
             }
           }
         ],
-        seriesParams: [{
-          show: 'true',
-          type: 'area',
-          mode: 'stacked',
-          data: {
-            label: 'Count',
-            id: '1'
-          },
-          drawLinesBetweenPoints: true,
-          showCircles: true,
-          interpolate: 'linear',
-          valueAxis: 'ValueAxis-1'
-        }],
+        seriesParams: [
+          {
+            show: true,
+            type: ChartTypes.AREA,
+            mode: ChartModes.STACKED,
+            data: {
+              label: 'Count',
```

Comment:
Probably unrelated, but shouldn't this be i18n-ified?

Reply:
It seems it should. I added localization.